ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/ObservedCveModeSelect.tsx

```diff
@@ -10,14 +10,10 @@ import {
 } from '@patternfly/react-core';
 import { SecurityIcon, UnknownIcon } from '@patternfly/react-icons';
 
-import { CRITICAL_SEVERITY_COLOR } from 'constants/severityColors';
 import { ObservedCveMode, isObservedCveMode, observedCveModeValues } from '../../types';
+import { getViewStateDescription, getViewStateTitle } from './string.utils';
 
-export const WITH_CVE_OPTION_TITLE = 'Image vulnerabilities';
-export const WITHOUT_CVE_OPTION_TITLE = 'Images without vulnerabilities';
-export const WITH_CVE_OPTION_DESCRIPTION = 'Images and deployments observed with CVEs';
-export const WITHOUT_CVE_OPTION_DESCRIPTION =
-    'Images and deployments observed without CVEs (results may be inaccurate due to scanner errors)';
+const width = '330px';
```

Comment:
this may be a good opportunity to use the screaming snake case 🐍  since it's a standard constant value? although i do see the convenience of being able to use the shorthand to refer to the key and value simultaneously 🤔

Reply:
Yeah this was primarily due to being able to use the shorthand.

I also think of the `width` constant conceptually as just a value that is part of this component, similarly to anything that would be declared as `const` within the component itself. That being opposed to some exported constant that is intended to be used throughout the code and convey `THIS_IS_A_CONSTANT_VALUE`. This is definitely subjective though.